Slingshot.Api/ng/Scripts/app.js

```diff
@@ -225,15 +234,21 @@ function IsSiteLocationParam(paramName) {
         }
 
         function getQueryVariable(variable) {
-            var query = window.location.search,
-                token = variable + "=",
-                startIndex = query.indexOf(token);
-
-            if (startIndex >= 0) {
-                return query.substring(startIndex + token.length);
```

Comment:
I can't even make sense of this old code. If I read it correctly, it only works if there is a single item on query string? Is that why you had to rewrite it?